packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -8727,49 +8727,9 @@ addEventListener('fetch', event => {});`
 
 			it("should support durable object bindings to SQLite classes with containers (dockerfile flow)", async () => {
 				vi.stubEnv("WRANGLER_DOCKER_BIN", "/usr/bin/docker");
-				function mockGetVersion(versionId: string) {
```

Comment:
The change here does not seem to be directly related to the PR, could you please add a note in the PR description?